package.json

```diff
@@ -23,6 +23,7 @@
     "builder": "~2.1.4",
     "builder-victory-component": "~0.0.5",
     "d3-scale": "^0.3.0",
+    "lodash": "^3.10.1",
```

Comment:
I'm going to remove that, but I think the `dependencies` should be declared in the current project `package.json` even if they are common for multiple project. What do you think?